pkg/providers/aws/cluster_network_provider.go

```diff
@@ -1510,18 +1515,61 @@ func getNetworkProviderConfig(ctx context.Context, configManager ConfigManager,
 		return nil, errorUtil.Wrap(err, "failed to unmarshal aws vpc create config")
 	}
 
-	if vpcCreateConfig.CidrBlock == nil {
-		return nil, errorUtil.New(fmt.Sprintf("CidrBlock in _network strategy tier %s can not be nil", tier))
+	// if the config map is found and the _network block contains an entry, that is returned for use in the network creation
+	if vpcCreateConfig != nil && vpcCreateConfig.CidrBlock != nil && *vpcCreateConfig.CidrBlock != "" {
+		_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+		if err != nil {
+			return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		}
+		cidrMask, _ := vpcCidr.Mask.Size()
+
+		logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
+		return vpcCidr, nil
+	}
+
+	//if vpcCreateConfig.CidrBlock is nil or an empty string we can go ahead and set a sensible non overlapping default with a mask size of /26
+	defaultCIDR, err := getNonOverlappingDefaultCIDR(ctx, n.Client, n.Ec2Api, n.Logger)
+	if err != nil {
+		return nil, errorUtil.Wrap(err, "failed to generate default CIDR")
 	}
 
-	_, vpcCidr, err := net.ParseCIDR(*vpcCreateConfig.CidrBlock)
+	// update the config map with the default cidr
+	// if no config map is found an error will be returned as this is required by cro to configure any aws resources
+	err = configManager.ReconcileNetworkStrategy(ctx, n.Client, defaultCIDR.String())
 	if err != nil {
-		return nil, errorUtil.Wrap(err, "failed to parse cidr block from _network strategy")
+		return nil, errorUtil.Wrap(err, "failed to reconcile strategy config map")
 	}
-	cidrMask, _ := vpcCidr.Mask.Size()
+	// a default cidr is generated and updated to the config map, that is returned for use in the network creation
+	return defaultCIDR, nil
+}
 
-	logger.Infof("found vpc cidr block %s/%d in network strategy tier %s", vpcCidr.IP.String(), cidrMask, tier)
-	return vpcCidr, nil
+// getNonOverlappingDefaultCIDR returns a non overlapping cidr block based on the OpenShift vpc cidr block
+// the default mask is /26
+// for other masks the user is required to provide their own via config
+func getNonOverlappingDefaultCIDR(ctx context.Context, client client.Client, ec2Api ec2iface.EC2API, logger *logrus.Entry) (*net.IPNet, error) {
+	clusterVpc, err := getClusterVpc(ctx, client, ec2Api, logger)
+	if err != nil {
+		errMsg := "failed to get cluster vpc for cidr block"
+		return nil, errorUtil.Wrap(err, errMsg)
+	}
+	clusterCIDR := *clusterVpc.CidrBlock
```

Comment:
might be worth adding a nil check here. Alternatively you can use `aws.StringValue(pointer_to_my_string)` it will return an empty string if the value is nil. Avoiding the nil pointer, but will still need to check the value.